src/inference/include/openvino/runtime/core.hpp

```diff
@@ -603,10 +603,8 @@ class OPENVINO_RUNTIME_API Core {
      * @param plugin_name Name of a plugin. Depending on platform, `plugin_name` is wrapped with shared library suffix
      * and prefix to identify library full name.
      * For example, on Linux platform, plugin name specified as `plugin_name` will be wrapped as `libplugin_name.so`.
-     * Plugin search algorithm:
-     * - If plugin is located in the same directory as OpenVINO runtime library, it will be used.
-     * - If no, plugin is tried to be loaded from paths pointed by PATH/LD_LIBRARY_PATH/DYLD_LIBRARY_PATH
-     *   environment variables depending on the platform.
+     * Note that plugin should be located in the same directory as OpenVINO runtime library
```

Comment:
I propose to add information about openvino-version folders

Reply:
Done, thanks!